src/sinol_make/commands/run/__init__.py

```diff
@@ -203,9 +203,59 @@ def compile(self, solution):
             return False
 
 
-    def execute_oiejq(self, command, result_file, output_file, answer_file, time_limit, memory_limit):
+    def check_output_diff(self, output_file, answer_file):
+        """
+        Checks if the output file and the answer file are the same.
+        Returns True if they are the same, False otherwise.
```

Comment:
```suggestion
        Checks whether the output file and the answer file are the same.
```
Instead of the comment about returning, it's simpler to add the information that this function is returning a boolean to its signature.